packages/icon/package.py

```diff
@@ -98,9 +96,21 @@ class Icon(AutotoolsPackage):
             description='Enable the Serialbox2 serialization')
 
     # Optimization Features:
+    variant('loop-exchange', default=True, description='Enable loop exchange')
```

Comment:
I think we need to have a strategy what variants are True by default.
This affects the build of Icon implicitly. I am thinking of BB, where I would like to have a rather stable environment.

Reply:
De facto we are currently using the same defaults as icon.
I think this makes for a good strategy. ICON users will get what they are used to. We can trust in their careful crafting of their defaults.

Reply:
@jonasjucker What's your opinion on that strategy?

Reply:
That sound reasonable. Let's stick to the ICON defaults. Wasn't aware this exists.

Reply:
Good point. I added a link to the defaults of ICON.